zerver/openapi/zulip.yaml

```diff
@@ -1363,6 +1367,14 @@ paths:
           type: boolean
         example: false
         required: false
+      - name: is_owner
+        in: query
+        description: |
+          Whether the target user is an owner.
```

Comment:
Let's consistently say "target user is an organization owner."  -- i.e. always put the "organization" word in there.